src/main/java/online/shop/dao/jdbc/ProductDaoJdbсImpl.java

```diff
@@ -0,0 +1,109 @@
+package online.shop.dao.jdbc;
+
+import java.sql.ResultSet;
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.sql.PreparedStatement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import online.shop.dao.ProductDao;
+import online.shop.exception.DataProcessingException;
+import online.shop.lib.Dao;
+import online.shop.model.Product;
+import online.shop.util.ConnectionUtil;
+
+@Dao
+public class ProductDaoJdbсImpl implements ProductDao {
+    @Override
+    public Product create(Product product) {
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            String query = "INSERT INTO product_table (name, price) VALUES (?, ?);";
```

Comment:
this naming is redundant, it is obvious that statements are related to the tables, fix according to comment from Bohdan